ontology/yaml/resources/HVAC/entity_types/SENSOR.yaml

```diff
@@ -120,3 +120,62 @@ SENSOR_FDPM:
     - local_air_temperature_sensor
     - battery_charge_percentage_sensor
     - signal_strength_percentage_sensor
+
+SENSOR_CO_CO2_ATM:
+  description: "Atmosphere CO and CO2 combined sensor with local temp and RH readings."
+  is_canonical: true
+  implements:
+  - /SENSOR
+  uses:
+  - thirtysecondrolling_average_zone_air_co_concentration_sensor
+  - zone_air_co_concentration_sensor
+  - zone_air_co2_concentration_sensor
+  - local_air_relative_humidity_sensor
```

Comment:
these are sensor clusters, and this particular RH and temp sensor are located right on the sensor to detect conditions at the CO/CO2 sensor heads. I suppose technically they can be zone temp sensors (in the same way that the CO and CO2 sensors are techincally measuring the space they are in), but they aren't measuring the zone so much as they are measuring the conditions at the sensor itself for calibration purposes.

Reply:
Would internal be more appropriate in this case?

Reply:
Instead of Local/Internal - what about "ROOM" (already defined as part of FACILITIES entity) or "SPACE"?

**Control vs Monitor**
From a FacOps system perspective, there exists a distinction between "zone" and "room/space" where "zone" implies an element of _control_ (i.e. HVAC terminal unit, lighting ballast, etc.) and room/space implies _measure_ (i.e. occ sensors for space utilization analysis, IAQ/IEQ sensors, light intensity, sound levels, etc). 

When I want to analyze data from _monitor_ devices, I want to know their relation to space (and a unique spaceId). For _control_ devices, that analysis generally flows upstream from the "zone" through various air and hydronic systems. In a future digital twin state, we'll want to map zone to space, as they won't always be 1:1 (i.e. a single tstat that serves multiple collab spaces).

Reply:
i think for simplicity we will rename them from local to `zone`; after discussing with Keith, we went down the rabbit hole of "well, technically, all sensors measure their temperatures 'locally'" and so for consistency we simply set reverted the `local` descriptor to `zone`. 

From the perspective of DBO, a zone is simply a defined region within a building; not necessarily a single room, it can be a collection of rooms or perhaps a subregion of a single room. The question of which rooms map to which zones can be handled elsewhere, particularly through connections: this zone connects to these rooms. The semantic distinction between `zone` versus `room` are legitimate, and we do have `room` as a defined subfield, but in practice it would necessitate that the modeler *know* that the sensor (or device its connected to) serves only a single room; zone is less specific, and therefore allows the modeler to assign it without having to care about whether it serves a single room or multiple.

To summarize: we will remove `local` and use `zone` instead; we will avoid `room` so that we don't have to care about whether the sensor really does apply to just a single room or not.